5 - Shor's Algorithm on logical cat qubits using the Qrisp framework.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "ebe8e678",
    "metadata": {},
    "source": [
-    "# Implementing Shor's Algorithm using the Qrisp framework on Cat Qubits\n",
+    "# Shor's Algorithm on logical cat qubits using the Qrisp framework\n",
```

Comment:
nit-pick: Can you write "algorithm" in lower case please? (same on the notebook file name)